mantidproject/mantid
# Add workarounds for MSVC with update 3

Applies a workaround to the python interface in order to circumvent a suspected bug with VS2015 and update 3.

**To test:**

Apply update 3 and check it compiles and the python tests run.

_Does not need to be in the release notes._

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
